### ISSUE TITLE
Train model in PyTorch for movie reviews dataset.

### ISSUE THREAD

**Issue by cwmeijer** [dianna-ai/dianna-exploration#59]
Script to train the model should be stored in repo
Resulting model (ONNX) saved in the surfdrive

- [x] Make script for preprocessing
- [x] Do hyperparameter search & save model

**Comment by loostrum** [comment]
Tutorial for generating a model + saving to ONNX for this dataset is available [here](https://github.com/NVIDIA/NeMo/blob/main/tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb).

**Comment by loostrum** [comment]
We have a model that reaches ~80% accuracy on the validation data, is this enough?

**Comment by loostrum** [comment]
Conclusion: yes, it is just a demo dataset and on another dataset a similar model reach 85% - similar enough.

**Comment by loostrum** [comment]
Selected model with lowest validation loss and uploaded ONNX file to surfdrive. The selected hyperparameters are from the run tagged on W&B.
